--- __main__.py 2025-01-12 12:08:32.699567820 -0500
+++ __main__.py.patched 2025-01-12 12:09:49.192874447 -0500
@@ -67,6 +67,16 @@
         help="Download latest voices.json during startup",
     )
     #
+    parser.add_argument(
+        "--use-cuda",
+        action="store_true",
+        help="Use GPU"
+    )
+    parser.add_argument(
+        "--espeak-data-dir",
+        help="Path to espeak-ng data directory"
+    )
+    #
     parser.add_argument("--debug", action="store_true", help="Log DEBUG messages")
     parser.add_argument(
         "--log-format", default=logging.BASIC_FORMAT, help="Format for log messages"
