--- handler.py  2025-01-04 19:50:00.000000000 +0000
+++ handler.py.patched  2025-01-04 20:58:52.201781636 +0000
@@ -88,7 +88,7 @@

             _LOGGER.debug("input: %s", input_obj)
             piper_proc.proc.stdin.write(
-                (json.dumps(input_obj, ensure_ascii=False) + "\n").encode()
+                (f"{text}\n").encode()
             )
             await piper_proc.proc.stdin.drain()
