--- __main__.py	2025-01-12 12:08:32.699567820 -0500
+++ __main__.py.patched	2025-01-12 12:13:42.368857466 -0500
@@ -53,6 +53,10 @@
     parser.add_argument(
         "--auto-punctuation", default=".?!", help="Automatically add punctuation"
     )
+    parser.add_argument(
+        "--sentence-silence",
+        help="Seconds of silence to add after each sentence"
+    )
     parser.add_argument("--samples-per-chunk", type=int, default=1024)
     parser.add_argument(
         "--max-piper-procs",
@@ -67,6 +71,16 @@
         help="Download latest voices.json during startup",
     )
     #
+    parser.add_argument(
+        "--use-cuda",
+        action="store_true",
+        help="Use GPU"
+    )
+    parser.add_argument(
+        "--espeak-data-dir",
+        help="Path to espeak-ng data directory"
+    )
+    #
     parser.add_argument("--debug", action="store_true", help="Log DEBUG messages")
     parser.add_argument(
         "--log-format", default=logging.BASIC_FORMAT, help="Format for log messages"
