--- process.py	2025-01-04 19:50:00.000000000 +0000
+++ process.py.patched	2025-01-04 20:58:10.813174885 +0000
@@ -130,7 +130,6 @@
                 str(config_path),
                 "--output_dir",
                 str(wav_dir.name),
-                "--json-input",  # piper 1.1+
             ]
 
             if voice_speaker is not None:
@@ -148,6 +147,12 @@
             if self.args.noise_w:
                 piper_args.extend(["--noise-w", str(self.args.noise_w)])
 
+            if self.args.espeak_data_dir:
+                piper_args.extend(["--espeak_data", str(self.args.espeak_data_dir)])
+
+            if self.args.cuda:
+                piper_args.extend(["--cuda"])
+
             _LOGGER.debug(
                 "Starting piper process: %s args=%s", self.args.piper, piper_args
             )
