--- process.py	2025-01-12 14:43:19.398599231 -0500
+++ process.patched	2025-01-12 14:44:38.795966270 -0500
@@ -148,6 +148,15 @@
             if self.args.noise_w:
                 piper_args.extend(["--noise-w", str(self.args.noise_w)])
 
+            if self.args.sentence_silence:
+                piper_args.extend(["--sentence-silence", str(self.args.sentence_silence)])
+
+            if self.args.espeak_data_dir:
+                piper_args.extend(["--espeak_data", str(self.args.espeak_data_dir)])
+
+            if self.args.use_cuda:
+                piper_args.extend(["--use-cuda"])
+
             _LOGGER.debug(
                 "Starting piper process: %s args=%s", self.args.piper, piper_args
             )
