--- process.py	2025-01-04 19:50:00.000000000 +0000
+++ process.py.patched	2025-01-04 20:58:10.813174885 +0000
@@ -148,6 +147,12 @@
             if self.args.noise_w:
                 piper_args.extend(["--noise-w", str(self.args.noise_w)])
 
+            if self.args.espeak_data_dir:
+                piper_args.extend(["--espeak_data", str(self.args.espeak_data_dir)])
+
+            if self.args.use_cuda:
+                piper_args.extend(["--use-cuda"])
+
             _LOGGER.debug(
                 "Starting piper process: %s args=%s", self.args.piper, piper_args
             )
