--- __main__.py	2025-01-04 21:57:16.229874190 +0000
+++ __main__.patched	2025-01-04 21:56:57.917558487 +0000
@@ -66,6 +66,16 @@
         help="Download latest voices.json during startup",
     )
     #
+    parser.add_argument(
+        "--use-cuda",
+        action="store_true",
+        help="Use GPU"
+    )
+    parser.add_argument(
+        "--espeak-data-dir",
+        help="Path to espeak-ng data directory"
+    )
+    #
     parser.add_argument("--debug", action="store_true", help="Log DEBUG messages")
     args = parser.parse_args()
 
